docs/config.md

```diff
@@ -0,0 +1,22 @@
+---
+id: config
+title: bolt config
+---
+
+---
+
+### **`bolt config delete <key>`**
+
+Removes the given `key` from the config.
+
+### **`bolt config list`**
+
+Displays the current configuration.
+
+### **`bolt config get <key>`**
+
+Prints the value for a given `key`.
+
+### **`bolt config set <key> <value> [-g|--global]`**
+
+Sets the config `key` to the given `value`.
```

Comment:
can we also add information that this will update yarn config

Reply:
Do you think it makes sense to add it to all of them? 

IMO, it is understandable and not needed. Just like what `yarn` does :)